test/performance/playwright.config.ts

```diff
@@ -14,9 +14,7 @@ process.env.ASSETS_PATH = path.join( __dirname, 'assets' );
 
 const config = defineConfig( {
 	...baseConfig,
-	reporter: process.env.CI
-		? './config/performance-reporter.ts'
-		: [ [ 'list' ], [ './config/performance-reporter.ts' ] ],
+	reporter: [ [ 'list' ], [ './config/performance-reporter.ts' ] ],
```

Comment:
I think we should merge it with this change as it now prints the errors correctly, which would allow us to close https://github.com/WordPress/gutenberg/issues/60366. What do you think @Mamaduka @kevin940726?

Reply:
Sounds good. The performance test results are now attached to the summary so they won't be lost in the noise (#61450).

Reply:
Ah, enabling the list reporter in this case actually means that only the error will be printed, not the tests themselves. It's because we're still using promisified `exec` and don't resolve with the `stdout` data. So, nothing in the output will change unless an error is thrown! 😄

Reply:
Good to know 😅

Reply:
Pinging @jsnajdr, since it also means we don't need to refactor to `spawn`! 😄